datacache/database.py

```diff
@@ -17,6 +17,10 @@
 
 from typechecks import require_integer, require_string, require_iterable_of
 
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
How do we externally configure the verbosity of logging output? I sometimes want to run a script with maximal debug logging but in other cases (e.g. Isaac sifting through 10mb of log messages) we might only want WARN.